devstack/plugin.sh

```diff
@@ -3,6 +3,7 @@
 function install_heat_infoblox {
     cd $HEAT_INFOBLOX_DIR
     sudo python setup.py install
+    sudo pip install -r requirements.txt
```

Comment:
What is the point to install the dependencies after the installation of the project?  I believe that ```sudo pip install -r requirements.txt``` should be executed previously to the ```sudo python setup.py install```.